src/utils/helpers_result_extraction.jl

```diff
@@ -0,0 +1,141 @@
+export extract_histories, extract_parameters, stack_extracts, wide_params, wide_histories
```

Comment:
note that the collection (a DataFrame) is being done from here on: https://github.com/EarthyScience/EasyHybrid.jl/blob/1736d03e9e734124b0e20e85ffae7ccd283c3a4b/src/train.jl#L338, for parameters, yes maybe we need some convenience functions. But, for DataFrames we could just show how use a normal `filter` for a `DataFrame`.